src/formatters/diffs/index.js

```diff
@@ -0,0 +1,38 @@
+import makeArraysDiff from './array';
+import makeObjectsDiff from './object';
+import makeStringsDiff from './string';
+import makeDatesDiff from './date';
+import truncateDiffs from './truncate-diff';
+
+export default class Diffs {
+    constructor (err, maxStringLength) {
+        this.expected = '';
+        this.actual   = '';
+        this.marker   = '';
+
+        this._buildDiff(err, maxStringLength);
+    }
+
+    _buildDiff (err, maxStringLength) {
+        var diffs = {
```

Comment:
I think we should just keep builder function instead of class. I thought we will have classes for all diffs and inheritance chain but it's not the true, from that I see.